Hymns/Home/HomeContainerView.swift

```diff
@@ -27,10 +28,13 @@ struct HomeContainerView: View {
                     .tabItem {HomeTab.settings.getImage(selectedTab == HomeTab.settings).imageScale(.large)}
                     .tag(HomeTab.settings)
                     .hideNavigationBar()
-            }.onAppear {
+            }.hideNavigationBar().padding(.top, self.iosAbove134 ? 5 : 0).onAppear {
```

Comment:
I'm confused here. So instead of removing the top padding for ios below 13.4, you're adding padding if it is ios 13.4 and above??

Reply:
Removing the top padding for ios below 13.4 does nothing. The hideNavigationBar helps though. The thing is I wanted to add padding for the iOS above 13.4 to help us in landscape if you see my screenshot. But that will then add more padding to iOS 13.3 which we obviously want to avoid since it has too much as is.

Reply:
I see. Could you take some before and after screenshots? I tried to remove the padding, and it looks fine to me...

Reply:
without padding:
<img width="1754" alt="Screen Shot 2020-06-15 at 9 50 38 PM" src="https://user-images.githubusercontent.com/1427524/84733165-56ffb880-af52-11ea-9087-c98a90de7bf5.png">
<img width="1756" alt="Screen Shot 2020-06-15 at 9 50 50 PM" src="https://user-images.githubusercontent.com/1427524/84733169-5a933f80-af52-11ea-8d06-e269d1b804ed.png">
<img width="1753" alt="Screen Shot 2020-06-15 at 9 51 04 PM" src="https://user-images.githubusercontent.com/1427524/84733171-5bc46c80-af52-11ea-9b3b-865e8991c9a1.png">

Reply:
with padding:
<img width="1749" alt="Screen Shot 2020-06-15 at 9 52 46 PM" src="https://user-images.githubusercontent.com/1427524/84733255-9c23ea80-af52-11ea-90f8-9b90a428851f.png">
<img width="1751" alt="Screen Shot 2020-06-15 at 9 52 56 PM" src="https://user-images.githubusercontent.com/1427524/84733264-a0500800-af52-11ea-87d9-fbc1d3247aea.png">
<img width="1757" alt="Screen Shot 2020-06-15 at 9 53 03 PM" src="https://user-images.githubusercontent.com/1427524/84733268-a1813500-af52-11ea-923d-fb7f409461fc.png">

Reply:
Lol when I did this using the same simulator and no padding I get the following. 
![Simulator Screen Shot - iPhone 8 - 2020-06-16 at 21 34 45](https://user-images.githubusercontent.com/15247726/84848768-639b1400-b019-11ea-98ec-b8376afdcf9c.png)
![Simulator Screen Shot - iPhone 8 - 2020-06-15 at 18 54 02](https://user-images.githubusercontent.com/15247726/84848770-6433aa80-b019-11ea-9ca3-2078e87edc1f.png)

Reply:
lol. Alright well let's just get this on with only the navigation bar, and then I'll create another PR with your changes if I run into it again. That way we can keep the PRs distinct anyway. makes it easier to search through in the future